ZAPD/OtherStructs/SkinLimbStructs.h

```diff
@@ -80,17 +80,17 @@ class Struct_800A598C : public ZResource
 	uint16_t unk_0;  // Length of unk_8
 	uint16_t unk_2;  // Length of unk_C
 	uint16_t unk_4;  // 0 or 1 // Used as an index for unk_C
-	segptr_t unk_8;  // Struct_800A57C0*
-	segptr_t unk_C;  // Struct_800A598C_2*
+	segptr_t unk_8;  // SkinVertex*
+	segptr_t unk_C;  // SkinTransformation*
```

Comment:
Most of these members could probably be named now.

Reply:
there's only one thing that wasn't named in decomp yet (``unk_4`` from ``SkinLimbModif``)

I also named the skin limb types thing in the enum since there's something in decomp for that, 4 and 11 values have a signification but other ones will make the segment ``NULL``, that's why the 5th one was removed (and applied formatter as always)

Reply:
I mean name the members in this class.